_articles/pop-keyboard-shortcuts.md

```diff
@@ -17,6 +17,20 @@ The Super Key is the OS key on your keyboard and is the Ubuntu or Pop key (<kbd>
 
 In Pop!\_OS, <kbd><span class="fl-pop-key"></span></kbd> handles the majority of OS and window actions.
 
+With the release of Pop!_OS 20.04 LTS there are new shortcuts with the introduction of Pop Shell and here are some of the shortcuts.
```

Comment:
With the release of Pop!_OS 20.04 LTS and Pop Shell, there are new keyboard shortcuts available: